### PR TITLE
feat: add collection data to top selling collection endpoint

### PR DESCRIPTION
**Pull request by pedromcunha**

### CODE REVIEW

File: packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts [hunk 1]
```diff
@@ -96,51 +121,163 @@ export const getTopSellingCollectionsOptions: RouteOptions = {
...
+      period,
+      fillType,
+      limit,
+      includeRecentSales,
+      normalizeRoyalties,
+      useNonFlaggedFloorAsk,
+    } = request.query;
 
     try {
+      let startTime = Math.floor(new Date().getTime() / 1000) - 60 * 24 * 60;
```
Comment (lunarmayor) [inline packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts:136]: can we save this to a variable `Math.floor(new Date().getTime() / 1000)` seems to be repeated a lot here

File: packages/indexer/src/elasticsearch/indexes/activities/index.ts [hunk 2]
```diff
@@ -205,7 +206,7 @@ const mapBucketToCollection = (bucket: any, includeRecentSales: boolean) => {
     id: collectionData?.id,
     name: collectionData?.name,
     image: collectionData?.image,
-    primaryContract: collectionData?.contract,
+    primaryContract: data?.contract,
```
Comment (lunarmayor) [inline packages/indexer/src/elasticsearch/indexes/activities/index.ts:209]: oh interesting guess this was never working?

Comment (pedromcunha) [reply packages/indexer/src/elasticsearch/indexes/activities/index.ts:209]: yeah it wasn't working, I patched it. We were just grabbing it from the wrong node.

File: packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts (hunk 1 above)
Comment (pedromcunha) [reply packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts]: done, good point!

File: packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts [hunk 3]
```diff
@@ -14,9 +14,9 @@ import {
 
 import { getJoiPriceObject, JoiPrice } from "@/common/joi";
 
-const version = "v5";
```
Comment (lunarmayor) [inline packages/indexer/src/api/endpoints/collections/get-top-selling-collections/v1.ts:17]: oops 😅

**Review by lunarmayor — APPROVED** [review]
looks good to me

**Pull request merged**